### PR TITLE
fix(components/forms): toggle switch `ariaLabel` input gives info on when to be contextual

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**
[AB#2581958](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2581958)

### CODE REVIEW

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts [hunk 1]
```diff
@@ -54,7 +54,8 @@ export class SkyToggleSwitchComponent
   /**
    * The ARIA label for the toggle switch. This sets the toggle switch's `aria-label`
```
Comment (blackbaud-johnly) [inline libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts]:
```suggestion
   * The ARIA label for the toggle switch. This sets the `aria-label`
```

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts [hunk 2]
```diff
@@ -54,7 +54,8 @@ export class SkyToggleSwitchComponent
...
    * attribute to provide a text equivalent for screen readers [to support accessibility](https://developer.blackbaud.com/skyux/learn/accessibility).
-   * If the `sky-toggle-switch-label` component displays a visible label, do not use this property.
+   * Use a context-sensitive label, such as "Annual fundraiser is active." Context is especially important when multiple toggle switches are in close proximity.
```
Comment (blackbaud-johnly) [inline libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts:57]: The "Annual fundraiser is active" only reflects one state for the toggle switch. So I think this ARIA label is incorrect half the time. I mean, if the toggle switch is in the unselected state but the ARIA label says "Annual fundraiser is active," I think that might be confusing to screen reader users.

Should we provide an example that works for both selected and unselected states? Or call out that both states need to be accounted for? I'm not sure how to do that without getting long-winded, but my first thought is to flesh this out to something like "Use a context-sensitive label. For example, when a toggle switch activates and deactivates an annual fundraiser, use "Annual fundraiser is active" or "Annual fundraiser is inactive" depending on the selected state." That's much longer, but I _think_ it's more accurate. Let me know if you want to coordinate further or if I need to pull in Matt Gregg.

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts [hunk 3]
```diff
@@ -54,7 +54,8 @@ export class SkyToggleSwitchComponent
...
+   * If the `sky-toggle-switch-label` component displays a visible label, use this property if extra context is needed due to that context only being available visibly.
```
Comment (blackbaud-johnly) [inline libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts]:
```suggestion
   * When the `sky-toggle-switch-label` component displays a visible label, this property is only necessary if that label requires extra context.
```
I don't feel like the second half of this sentence was very clear. I think we need "only" in there to convey that using this property when there is a visible label is an exception to the rule. And I don't understand what we mean by "context only being available visibly" given that it's presumably the label that should provide context and screen readers will pick it up. So I rewrote to add "only" and work around the bit at the end. Do you think this rewrite works?

File: libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts (hunk 2 above)
Comment (blackbaud-johnly) [inline libs/components/forms/src/lib/modules/toggle-switch/toggle-switch.component.ts]:
```suggestion
   * Use a context-sensitive label, such as "Activate annual fundraiser" for a toggle switch that activates and deactivates an annual fundraiser. Context is especially important if multiple toggle switches are in close proximity.
```

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1806/storybook/)

Component Storybooks:

- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/colorpicker-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/errors-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/forms-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/lookup-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/phone-field-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1806/storybooks/theme-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1806/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1806/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1806/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1806/playground/)

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**